### ISSUE TITLE
Provide way to expand full collection names in damsmanager UI

### ISSUE THREAD

**Issue by remerjohnson** [ucsdlib/damsmanager#268]
### Descriptive summary

We need a way to fully expand collection names where they exist in the UI (i.e., the Collection Import/Create&Edit tools, the Excel Import tool, etc.). This is the current view of two CalCOFI collections, one which has a "2017" at the end but is obscured:

![calcofi_cutoff_text](https://user-images.githubusercontent.com/5183580/46623773-9de1ad80-cae3-11e8-8f5f-092a0b9f2689.PNG)

### Rationale

When collection names are long enough, the current abbreviated text can appear the same, and the "correct" collection is impossible to determine (see screenshot). 

### Related work

ucsdlib/dams-metadata/issues/27 
[LiSN discussion of UI cutoff problem](https://lisn.ucsd.edu/x/N4RrB)

**Comment by lsitu** [comment]
@gamontoya Sure. I'll start it once I get ticket [Check for recent bot crawls ](https://github.com/ucsdlib/damspas/issues/530) done.
@remerjohnson Please let me know if you are waiting on it.

**Comment by remerjohnson** [comment]
@lsitu Nope, this is not a critical fix, but would be nice to have next sprint. Thanks

**Comment by lsitu** [comment]
@remerjohnson No problem. I am working on it now.

**Comment by lsitu** [comment]
@mcritchlow I've created PR https://github.com/ucsdlib/damsmanager/pull/272 to expend the full collection names in the selection option. It's ready for review now. Thanks.
@remerjohnson Now when you move the the mouse over the select option, the full collection name will be shown.

**Comment by remerjohnson** [comment]
@lsitu That sounds like a great option. Thanks

**Comment by lsitu** [comment]
@mcritchlow Sorry I created PR https://github.com/ucsdlib/damsmanager/pull/277 to fix a mistake in my last commit that breaks the control panel page. It's ready for review now. Thanks.
